View/ViewHandler.php

```diff
@@ -254,18 +258,26 @@ protected function getSerializationContext(View $view)
     {
         $context = $view->getSerializationContext();
 
-        if ($context->attributes->get('groups')->isEmpty() && $this->exclusionStrategyGroups) {
+        if($context instanceof Context) {
+            $JMSAdapter = new JMSContextAdapter();
+            $context = $JMSAdapter->transform($context);
+        }
+
+        if (empty($context->getGroups()) && $this->exclusionStrategyGroups) {
```

Comment:
I didn't work on the compatibility yet.
I wanted before to submit an idea to resolve the problem of the 2.7 serializer.

Do you agree globally with the new classes ?